src/script_functions.c

```diff
@@ -2102,16 +2010,14 @@ int c_set_viewport(lua_State *lua)
  */
 int c_center(lua_State *lua)
 {
+	if (!check_param_counts_range(lua, "center", 0, 2)) {
+		return 0;
+	}
+
 	int top = lua_gettop(lua);
 
 	GdkRectangle desktop_r, window_r;
 
-	if (top > 2) {
-		// no input, or one input
-		luaL_error(lua, "center: %s", one_or_two_indata_expected_error);
```

Comment:
This was the wrong error message since zero was also an option, but the prospect of adding another unique string for zero_or_one_or_two is what convinced me to just add the n_to_m message instead.